prisma/schema.prisma

```diff
@@ -23,6 +23,74 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+// Achhievement
+// A final achievement can consist of several under achievements
+model achievement_template {
+  id                            Int                       @id @default(autoincrement())
+  metrics                       String[]                  @default([])
+  templateFor                   template_for_enum
+  // Groups defines the "final-achievement"
+  group                         String                    @unique @db.VarChar
```

Comment:
I think, unfortunately, we cannot make the field unique. It would prevent us from creating multiple achievements with the same group name.